src/backend/renderer/multigpu/mod.rs

```diff
@@ -1310,6 +1381,7 @@ where
     <T::Device as ApiDevice>::Renderer: ImportDma + ImportMem,
     <<R::Device as ApiDevice>::Renderer as Renderer>::Error: 'static,
     <<T::Device as ApiDevice>::Renderer as Renderer>::Error: 'static,
+    Target: Clone,
```

Comment:
What introduced this Clone bound btw?

Reply:
Because we cache the renderbuffer and `Bind<Target>` consumes the buffer otherwise.